### PR TITLE
Post

### PR DESCRIPTION
**Pull request by house9**

### CODE REVIEW

File: post.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# How to add TailwindCSS to Ruby on Rails
+
+TailwindCSS, when paired with Rails applications, offers numerous benefits that significantly elevate both the developer experience and the end-user interface. This article provides a comprehensive guide on integrating TailwindCSS into your Rails application.
```
Comment (nfuller52) [inline post.md:3]: Last sentence is redundant of the article title

**Review by nfuller52 — COMMENTED** [review]
Why did you mention setting up tailwind with npm if you don't review how to do it that way?

It feels "left out". I would either remove the `Ruby Gem or NPM Package?` part or add in how to install tailwind with npm. Which could be a ball of wax adding webpack/vite and all the things...

I feel like the digression about `overmind` is interesting. Would be nice to see why that's bertter than Foreman?

File: post.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+```bash
+rails generate scaffold Post title:string content:text
+```
+
+The views generated will be styled with Tailwind out of the box!
+
+## Custom Tailwind in the sample application
+
+In the sample application, there's a custom Tailwind utility for styling radio buttons. Located in `app/assets/stylesheets/application.tailwind.css`, there's a custom class that makes use of the CSS `:has` pseudo-class for radio button styling. It's important to note that [:has](https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility) is experimental and not supported in all browsers.
```
Comment (nfuller52) [inline post.md]: This feels oddly out of place? What's the purpose for calling it out? What does this have to do with "How to add TailwindCSS to Ruby on Rails"?

File: post.md (hunk 1 above)
Comment (sarmstead) [reply post.md:3]: We plan to leave that last line in for SEO 🪄

File: post.md (hunk 2 above)
Comment (sarmstead) [reply post.md]: Good call! Removing it from this post. We may write a separate post about this 🤷‍♂️

**Comment by sarmstead** [comment]
> Why did you mention setting up tailwind with npm if you don't review how to do it that way?
> It feels "left out". I would either remove the `Ruby Gem or NPM Package?` part or add in how to install tailwind with npm. Which could be a ball of wax adding webpack/vite and all the things...

@nfuller52, good call here. We switched the order around and made it more clear that we will only cover the Ruby Gem setup. We also added links to Tailwind's docs on NPM setup. 

> I feel like the digression about `overmind` is interesting. Would be nice to see why that's bertter than Foreman?

We had a good link in the Resources section. We added that to the main content area to give readers the option to checkout the post we referenced for that section.
